bids/layout/layout.py

```diff
@@ -937,6 +937,14 @@ def get(self, return_type='object', target=None, scope='all',
 
         entities = self.get_entities()
 
+        # error check on users accidentally passing in filters
+        if 'filters' in filters:
+            if isinstance(filters['filters'], dict):
+                raise RuntimeError('You passed in filters as a dictionary '
+                                   'named filters; please pass the keys in '
+                                   'as named keywords to the `get()` call. '
+                                   'For example: `layout.get(**filters)`.')
```

Comment:
We can be slightly briefer, by letting `dict.get()` return `None` if it's missing:

```suggestion
        if isinstance(filters.get('filters'), dict):
            raise RuntimeError('You passed in filters as a dictionary named '
                               'filters; please pass the keys in as named '
                               'keywords to the `get()` call. For example: '
                               '`layout.get(**filters)`.')
```